diff --git a/pyVmomi/DynamicTypeManagerHelper.py b/pyVmomi/DynamicTypeManagerHelper.py
index 459da95..23f2604 100644
--- a/pyVmomi/DynamicTypeManagerHelper.py
+++ b/pyVmomi/DynamicTypeManagerHelper.py
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # VMware vSphere Python SDK
 # Copyright (c) 2008-2014 VMware, Inc. All Rights Reserved.
 #
diff --git a/pyVmomi/Iso8601.py b/pyVmomi/Iso8601.py
index 1f617d4..f759f61 100644
--- a/pyVmomi/Iso8601.py
+++ b/pyVmomi/Iso8601.py
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # VMware vSphere Python SDK
 # Copyright (c) 2008-2014 VMware, Inc. All Rights Reserved.
 #
diff --git a/pyVmomi/ManagedMethodExecutorHelper.py b/pyVmomi/ManagedMethodExecutorHelper.py
index bbde5d4..b49caf8 100644
--- a/pyVmomi/ManagedMethodExecutorHelper.py
+++ b/pyVmomi/ManagedMethodExecutorHelper.py
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # VMware vSphere Python SDK
 # Copyright (c) 2008-2014 VMware, Inc. All Rights Reserved.
 #
diff --git a/setup.py b/setup.py
index b1c5935..0e2942e 100644
--- a/setup.py
+++ b/setup.py
@@ -38,8 +38,6 @@ setup(
    packages=['pyVmomi', 'pyVim'],
    install_requires=required,
    license='License :: OSI Approved :: Apache Software License',
-   data_files = [('', ['LICENSE.txt', 'NOTICE.txt', 'MANIFEST.in',
-                       'setup.py', 'tox.ini', 'setup.cfg'])],
    classifiers=[
       'Development Status :: 5 - Production/Stable',
       'License :: OSI Approved :: Apache Software License',
@@ -62,8 +60,8 @@ setup(
       'Operating System :: MacOS',
    ],
    keywords='pyvmomi vsphere vmware esx',
-   platforms = ['Windows', 'Linux', 'Solaris', 'Mac OS-X', 'Unix'],
+   platforms=['Windows', 'Linux', 'Solaris', 'Mac OS-X', 'Unix'],
    test_suite='tests',
-   tests_require= required_for_tests,
+   tests_require=required_for_tests,
    zip_safe=True
 )
